utils/bigclown/bigclown-mqtt2influxdb/files/config.yml

```diff
@@ -0,0 +1,60 @@
+mqtt:
+  host: 127.0.0.1
+  port: 1883
+
+influxdb:
+  host: 127.0.0.1
+  port: 8086
+  database: node
+
+points:
```

Comment:
again, I'd suggest that this is not really a suitable default configuration, but could be shipped as an example perhaps.

Reply:
`points` are petty much sensible. Those are not most of the times changed unless you edit firmware of modules. With default firmware it makes sense to have exactly this settings. Problem is more with connection settings. I commented those values which should prevent service from starting. That way user has to review config and set connection settings to appropriate values.